apps/pie-microsite/src/content/pages/foundations/spacing/global.md

```diff
@@ -0,0 +1,13 @@
+---
+eleventyNavigation:
+    key: Global tokens
+    parent: Spacing
+    order: 2
+---
+## Our global spacing scale
+We have defined a series of global tokens to define our various spacing measures. The global spacing scale works in increments of 8px and 4px.
+
+{% tokensTable {
+    path: 'theme.jet.spacing.global',
+    prefix: 'spacing'
+} %}
```

Comment:
I think we should remove this until we pick up the ticket(s) to properly fill out this page